subsys/bluetooth/host/gatt.c

```diff
@@ -5140,6 +5153,57 @@ int bt_gatt_store_ccc(uint8_t id, const bt_addr_le_t *addr)
 	return 0;
 }
 
+uint8_t convert_to_public_on_match(const struct bt_gatt_attr *attr, uint16_t handle,
+			void *user_data)
+{
+	struct _bt_gatt_ccc     *ccc;
+	struct device_addresses *device_addresses   = user_data;
```

Comment:
There seems to be some white space issues here: remove the multiple consecutive space characters